script.js

```diff
@@ -1,3 +1,7 @@
+const SECOND = 60000; // milliseconds
```

Comment:
Shouldn't it be called "MINUTE" ?

Reply:
Of course, I think I was a bit too fast with the renaming